addons/mail/static/src/models/message/message.js

```diff
@@ -442,6 +442,33 @@ function factory(dependencies) {
                 this.originThread.isModeratedByCurrentPartner
             );
         }
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsSubjectEqualThreadName() {
+            if (!this.subject || !this.originThreadName) {
```

Comment:
Use `originThread.name` everywhere, instead of the related field which is only used for the dependency.